### PR TITLE
Add some functions related to mounts and ornaments.

### PR DESCRIPTION
**Pull request by Ottermandias**

### CODE REVIEW

File: FFXIVClientStructs/FFXIV/Client/Game/Character/Character.cs [hunk 1]
```diff
@@ -198,14 +202,17 @@ public struct ForayInfo
...
 	    [FieldOffset(0x08)] public BattleChara* OwnerObject;
 	    [FieldOffset(0x10)] public Character* MountObject;
 	    [FieldOffset(0x18)] public ushort MountId;
 	    [FieldOffset(0x1C)] public float DismountTimer;
 	    //1 in dismount animation, 4 = instant delete mount when dismounting (used for npcs and such)
 	    [FieldOffset(0x20)] public byte Flags;
 	    [FieldOffset(0x24)] public fixed uint MountedObjectIds[7];
+
+        [MemberFunction("E8 ?? ?? ?? ?? 48 8B 43 ?? 80 B8 ?? ?? ?? ?? ?? 74 ?? 0F B6 90")]
+        partial void SetupMount(short mountId, uint unk1, uint unk2, uint unk3, byte unk4);
```
Comment (Haselnussbomber) [inline FFXIVClientStructs/FFXIV/Client/Game/Character/Character.cs:215]: Missing `public` accessor here ;)

File: ida/data.yml [hunk 2]
```diff
@@ -3798,6 +3798,7 @@ classes:
       0x140742C00: IsMounted
       0x140744450: GetCompanionOwnerID
       0x1407453F0: GetTargetId
+      0x140745FB0: GetParentCharacter
```
Comment (wolfcomp) [inline ida/data.yml:3801]: Changed to `0x1407461A0`

File: ida/data.yml [hunk 3]
```diff
@@ -3823,6 +3824,9 @@ classes:
       - ea: 0x1419C5FC8
     funcs:
       0x1412C27A0: HasOrnament
+  Client::Game::Character::MountContainer:
+    funcs:
+      0x1412BE7B0: SetupMount
```
Comment (wolfcomp) [inline ida/data.yml:3829]: Changed to `0x1412BF010`

File: ida/data.yml [hunk 4]
```diff
@@ -6844,6 +6848,7 @@ classes:
         base: Client::Game::Character::Character
     funcs:
       0x1412D36D0: ctor
+      0x1412D3880: SetupOrnament
```
Comment (wolfcomp) [inline ida/data.yml:6851]: Changed to `0x1412D40E0`

**Review by Haselnussbomber — APPROVED** [review]

**Comment by Haselnussbomber** [comment]
I don't know how to mark my review as complete. I clicked approve, but it's till there. Sorry if I did it wrong. 😅

**Comment by Ottermandias** [comment]
Rebased, incorporated new addresses.

**Pull request merged**